client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceConfigModal.js

```diff
@@ -16,7 +16,7 @@ import css from './StartInstanceConfigModal.less';
 
 import {
   TextInput
-} from '../shared/components';
+} from '../../../app/primitives';
```

Comment:
Proposal: Find a better name for that folder. Do not move folder under `app` but `shared/sth-sth`. Maybe `shared/ui`?

I'd like to prevent to put more things into the app.

Reply:
If these are meant to be shared components, we'd need tests for them, too.

Reply:
At least test their existence.

Reply:
What about other primitives like [modals](https://github.com/camunda/camunda-modeler/tree/develop/client/src/app/primitives/modal), (dropdown) buttons, ...? We also use them outside the scope of the app.

Maybe we should move all of them outside of "primitives" but inside a shared folder called `client/shared/ui`.

Reply:
I think they are all UI components, so `shared/ui` sounds like a perfect dir.

Reply:
Makes sense.

Reply:
Would be a good change, overall. Putting all generic UI (only) building blocks in a single place would benefit the code base I believe.

Reply:
🚀